### ISSUE TITLE
InfoMapper does not load on Edge

### ISSUE THREAD

**Issue by smalers** [OpenWaterFoundation/owf-app-infomapper-ng#282]
A user reported that the Poudre application does not load when using Edge.  The following is the console message:

![image](https://user-images.githubusercontent.com/6448274/93379174-b81d7d80-f81a-11ea-86d1-219c8015391e.png)

**Comment by smalers** [comment]
I ran with edge and it loaded fine.  My Edge version is:  Version 85.0.564.51 (Official build) (64-bit)

Based on other research, it looks like older versions of Edge do not support features that are required by InfoMapper.  I will let Josh add specifics.  Consequently, the solution is to update Edge, perhaps also doing a Windows 10 update which will update Edge.

**Comment by Nightsphere** [comment]
I'm not sure if I did something different, but updating Windows did not update Edge for me, and did not fix the issue either. Going straight to https://www.microsoft.com/en-us/edge to download the new Edge successfully replaced the legacy version I had previously installed and now the InfoMapper is working just fine.

I've also learned that the legacy version of Edge I had before and Internet Explorer will both stop being supported by August 17th, 2021, and Microsoft has already been suggesting user's upgrade to the new Edge now.

This has been added to the list to add to troubleshooting in the dev docs. Closing the issue.
